lib/operations/create_indexes.js

```diff
@@ -15,7 +15,11 @@ const validIndexOptions = new Set([
   'expireAfterSeconds',
   'storageEngine',
   'collation',
-  'bucketSize'
+  'bucketSize',
```

Comment:
so it looks like for some reason we're missing a whole [bunch](https://github.com/mongodb/node-mongodb-native/blob/master/src/operations/indexes.ts#L13-L42) of options. My guess is that the old code for this would just merge in whatever options were provided, so if we're going to be strict we'll need to backport the list from 4.0

Reply:
updated!